common/djangoapps/course_modes/views.py

```diff
@@ -226,3 +228,31 @@ def _get_requested_mode(self, request_dict):
             return 'honor'
         else:
             return None
+
+
+@ensure_csrf_cookie
+@require_level('staff')
```

Comment:
@stephensanchez, this view is just for the bok-choy tests. We wanted to add Honor mode to a course to make it a PaidCourseRegistration to test out the functionality. This has nothing to do with the actual functionality of this story - just for testing. We originally had this view and URL under a testing flag but that was causing some other errors on jenkins. Can you suggest a better solution?